src/components/pagination/PaginationButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface Props {
+  isDisabled?: boolean;
+  isActive?: boolean;
+  onClick: (e: unknown) => void;
```

Comment:
Shouldn't this be a click event instead of unknown? Maybe you can use `Pick` to get it from the chakra button props